starwars/src/components/StarCards.js

```diff
@@ -0,0 +1,26 @@
+import React, {useState, useEffect}  from 'react';
+import CharCards from "./CharCards";
+import axios from 'axios';
+import { CardDeck } from "reactstrap";
+import "./StarWars.css";
+
+export default function StarCards() {
+    const[starPeople, setStarPeople] = useState([]);
+
+    useEffect(() => {
+        axios.get('https://swapi.co/api/people/').then(res => {
+            setStarPeople(res.data.results);
+            console.log(res.data);
+            console.log(res);
+        })
+        .catch(err => {
+            console.log('not working',err);
+        })
+    },[]);
+
+    return (
+        <CardDeck className='container'>
+            {starPeople.map(person => CharCards(person))}
```

Comment:
Using CharCards as the function it truly is, was interesting and cool to see! I would recommend using the component syntax for it just to make sure the rest of your team understands it's a child _component_.